releases.md

```diff
@@ -0,0 +1,106 @@
+# Express Release Process
+
+This document contains the technical aspects of the Express release process. The
+intended audience is those who have been authorized by the Express Technical
+Committee (TC) to create, promote and sign official release builds for Express,
+as npm packages hosted on https://npmjs.com/package/express.
+
+## Who can make releases?
+
+Release authorization is given by the Express TC. Once authorized, an individual
+must have the following:
+
+### 1. Github release access
+
+Releases can be tagged by members of the @expressjs/express team with Write or
+Admin permission level.
+[tunniclm: Is there  a separate way to indicate whether an individual is allowed
+           to tag a release, or some higher level of permission?]
+
+### 2. npmjs.com release accesss
+
+Releases can be published to npmjs.com by members of the npmjs.com express
+organisation.
+
+## How to create a release
+
+### 1. Check all the code and doc changes are in
+
+Check the milestone and labels for the release in github, depending on the
```

Comment:
I'm not sure if this was the way it has ever been done. Typically there are two main release flows: patch and non-patch.

The patch flow is pretty simple: it's just extremely simple updates, like typo fixes, patch dependency updates, and maybe a fix, depending on how risky it is. Every other change is always a non-patch flow.

The non-patch flow is done using a separate branch (for example `4.13`, `5.0`, etc.) and tracked using a release pull request (for example, https://github.com/expressjs/express/pull/2682, https://github.com/expressjs/express/pull/2237, etc.).